echo-sql/docker-compose.yml

```diff
@@ -1,7 +1,19 @@
-version: '3.7'
+version: "3.9"
 services:
-    postgres:
+  go-app:
+    build:
+      context: .
+    container_name: echoApp
+    ports:
+      - "8082:8082"
+    depends_on:
+      - postgres
+    networks:
+      - babab
```

Comment:
It's better to keep the network :- `keploy-network`. as most people would already have with same since we have mentioned same in our docs as well

Reply:
@Sonichigo I think it's fine to add a variety.